scripts/build-themes.js

```
@@ -0,0 +1,147 @@
+    ...backgroundColors,
+    ...transparentBackgroundColors
+  };
+
+  const additionalColors = variantConfig.additionalColors || {};
+  theme.colors = { ...theme.colors, ...additionalColors };
+
+  // Apply token color overrides if specified
+  if (variantConfig.tokenColorOverrides) {
+    for (const [scope, color] of Object.entries(variantConfig.tokenColorOverrides)) {
+      const tokenIndex = theme.tokenColors.findIndex(token => 
+        token.scope && token.scope.includes(scope)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)